src/main/java/org/openstreetmap/atlas/geography/Location.java

```diff
@@ -81,8 +81,8 @@ public static Location forStringLongitudeLatitude(final String locationString)
         {
             throw new CoreException("Invalid Location String: {}", locationString);
         }
-        final double latitude = Double.valueOf(split.get(1));
```

Comment:
You need to be a little careful with using parseDouble functions. They can throw NumberFormatExceptions if the string you are parsing is not valid. And it seems like you don't handle that exception in this function and the one above. I would suggest throwing it from the function.

Reply:
This is certainly true. In fact, the original `Double.valueOf` call had the same problem, if we check out the implementation in `java.lang.Double`

```
public static Double valueOf(String s) throws NumberFormatException {
        return new Double(parseDouble(s));
}
```

I am a little hesitant about declaring `NumberFormatException` to be thrown at the function level, since it is a `RuntimeException`, and in general we avoid declaring those explicitly in the `Atlas` codebase (clutters the function signature). However, in this case it may be worth it, since it would be possible for the caller to recover from invalid input.